receiver/main.py

```
@@ -42,6 +42,21 @@
 STATS_INTERVAL_MINUTES = 15     # Log stats every N minutes
 
+
+def _env_int(name: str, default: int) -> int:
+    """Read an integer env var with safe fallback."""
+    raw = os.environ.get(name)
+    if raw is None:
+        return default
+    try:
+        value = int(raw)
+        return value if value > 0 else default
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)